### ISSUE TITLE
TypeError: Cannot read property 'forEach' of undefined

### ISSUE THREAD

**Issue by pelotom** [Quramy/typed-css-modules#61]
After upgrading from 0.3.7 to 0.4.0 I'm getting this error message for each file processed by `tcm`:

```
[Error] TypeError: Cannot read property 'forEach' of undefined
```

I've reproduced this in a fresh project with only the following file, `src/test.css`:

```css
.hello {
  background-color: red;
}
```

then running `tcm src` I get the error:

```
Wrote /Users/tom/code/foo/src/test.css.d.ts
[Error] TypeError: Cannot read property 'forEach' of undefined
```

If more CSS files are added, an error is printed after processing each one.

**Comment by Quramy** [comment]
:bow:

It'll be fixed via https://github.com/Quramy/typed-css-modules/pull/60.

I've published #60 as v0.4.2

**Comment by pelotom** [comment]
Awesome, thanks for the fast turnaround!
